src/pattern-library/components/patterns/LayoutPatterns.js

```diff
@@ -9,12 +10,144 @@ import {
   PatternExample,
 } from '../PatternPage';
 
+function SquareBlock() {
```

Comment:
I will type/comment these demo components once the overall direction is approved.

Reply:
...And now done.